lib/plugins/aws/index.js

```diff
@@ -101,6 +101,14 @@ class SDK {
       return found.PhysicalResourceId;
     });
   }
+
+  getAccountId(iamRoleArnLambda) {
```

Comment:
You're right, it will no longer work. Reading the discussion on #1540 I'm a bit confused whether we still want to display the account Id, and if so then why not add it to outputs as suggested there?

I'm removing it for now to finish the merge.